toml-esm.mjs

```diff
@@ -0,0 +1,2 @@
+export {default as parse} from './parse.js';
+export {default as stringify} from './stringify.js';
```

Comment:
```suggestion
import TOML from './toml.js';

export const parse = TOML.parse;
export const stringify = TOML.stringify;
export default TOML;
```

Rely on module API instead of file paths since this is a wrapper.

(@LJNeon If this package ever gets an ESM rewrite, the way you had it is preferable.)

Also, include a default export in addition to named exports. Not sure this matters too much, since you can `import * as TOML...`, but there might be edge cases where it'll be better to have than not.

Reply:
I'm unsure of any relevant difference here, the existing code should function as a wrapper properly. Including a default export doesn't matter, those edge cases if they do exist are already covered by [existing Node features](https://nodejs.org/api/modules.html#modules_module_createrequire_filename), so I don't think I'll add it. Thanks for the suggestion though!